src/adagrams.js

```diff
@@ -1,15 +1,131 @@
+let LETTERS = {
```

Comment:
```suggestion
const LETTERS = {
```

I do like this for generating the pool